web/src/hooks/store-hooks.ts

```diff
@@ -41,18 +41,43 @@ export function useIsSubscribed() {
   return isSubscribed;
 }
 
-export function useStickyState(defaultValue: any, key: string) {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function useLocalStorageState(defaultValue: any, key: string) {
   const [value, setValue] = useState(() => {
-    const stickyValue = window.localStorage.getItem(key);
+    if (!key) {
+      return defaultValue;
+    }
+
+    try {
+      const storedValue = window.localStorage.getItem(key);
 
-    return stickyValue !== null ? JSON.parse(stickyValue) : defaultValue;
+      // no stored value
+      if (storedValue === null) {
+        return defaultValue;
+      }
+
+      return JSON.parse(storedValue);
+    } catch (e) {
+      // silently fail if no localStorage or JSON parse fails
+      return defaultValue;
+    }
   });
 
   useEffect(() => {
+    if (!key) {
+      return;
+    }
+
     try {
+      // remove item from storage if it's the default option
```

Comment:
If we were to update the default values in our code and we don't remove here we could have instances where a client is out of sync and think it's not at default.